test/suites/clustering.sh

```diff
@@ -3521,6 +3521,15 @@ test_clustering_groups() {
   # delete cluster group "newgroup"
   lxc cluster group delete cluster:newgroup
 
+  # Try to create a cluster group using yaml
+  lxc cluster group create cluster:node1 <<EOF
+name: newgroup-yaml
```

Comment:
this shouldnt work as resource.name is used for cluster group name, not the stdin data.
Also this breaks with the convention of not allowing overriding command line args with stdin data that the other commands have.